src/client/app/helpers/strings.ts

```diff
@@ -112,6 +112,11 @@ export class Strings {
 
     static readonly deleteSnippetConfirm = 'Are you sure you want to delete this snippet?';
 
+    static readonly triggerActionId = 'trigger-suggest';
```

Comment:
No no.  Please only include actual localizable strings in here (only the `triggerActionLabel`).  The rest should be constants in `editor.ts`, but it will never be localized or even make sense form here, so no need to be here in Strings.